raiden/network/transport/matrix/client.py

```diff
@@ -245,6 +245,13 @@ def get_aliases(self, room_id: str) -> Dict[str, Any]:
     def __repr__(self) -> str:
         return f"<GMatrixHttpApi base_url={self.base_url}>"
 
+    def disable_push_notifications(self) -> Dict[str, Any]:
+        return self._send(
+            "PUT",
+            "/pushrules/global/override/.m.master.rule/enabled/",
```

Comment:
Looks like a typo?
```suggestion
            "/pushrules/global/override/m.master.rule/enabled/",
```

Reply:
looks like but it's not ;) server-default push rules start with a `.`. Here is the [spec](https://matrix.org/docs/spec/client_server/r0.6.0#predefined-rules)